src/Components/App/App.js

```diff
@@ -48,6 +49,16 @@ class App extends Component {
     return date.split('-')[0];
   }
 
+  fetchCharacters = (characters) => {
+    let promises = [];
+    for (let i = 0; i < 10; i++) {
+      fetch(characters[i])
+        .then(res => res.json())
+        .then(character => promises.push(character))
+    }
+    return promises;
+  } 
+
```

Comment:
HECK YESSSSS LESSSGOOOOOOO!

So psyched we figured this out!  We were way overcomplicating it and putting the Promise.all in the wrong place the whole time.  Hooray!